src/redux/slices/dappModalsSlice.ts

```diff
@@ -0,0 +1,80 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { DappModalConfig } from '../../UI/DappModal/types';
+import { logoutAction } from '../commonActions';
+
+export interface ModalsSliceState {
+  visible: boolean;
+  modal: DappModalConfig;
+}
+
+const initialState: ModalsSliceState = {
+  visible: false,
+  modal: {
```

Comment:
is nesting necessary? could be resolved with namespacing like: `showModalHeader` instead of `modal.showHeader`

Reply:
It's better to keep the modal configuration group in an object. If is needed to select showModalHeader then a simple selector should be added otherwise you will be forced to create a new selector for each new property added. Imagine how will look like when this modal will be fully customizable.